pygsti/protocols/rb.py

```diff
@@ -1273,17 +1410,9 @@ def _get_rb_fits(circuitdata_per_depth):
             adj_sps = []
             for depth in depths:
                 percircuitdata = circuitdata_per_depth[depth]
-                #print(percircuitdata)
-                if self.square_mean_root:
-                    #print(percircuitdata)
-                    adj_sps.append(_np.nanmean(_np.sqrt(percircuitdata))**2)
-                    #print(adj_sps)
-                else:
-                    adj_sps.append(_np.nanmean(percircuitdata))  # average [adjusted] success probabilities or energies
+                adj_sps.append(_np.nanmean(percircuitdata))  # average [adjusted] success probabilities or energies
```

Comment:
The removal of the `square_mean_root` kwarg (and related behavior shown here) is unrelated to the IRB stuff and was done at the recommendation of @tjproct. This was added at one point as an experiment and no longer makes sense to keep as an analysis option.